File: ParcialWeb/ClientApp/src/app/Emergencia/models/persona.ts
import { Vacuna } from "./vacuna";

export class Persona {
    tipoDocumento: string;
    cedula: string;
    nombre: string;
    apellido: string;
    edad: number;
    fechaNacimiento: string;
    institucionEducativa: string;
    nombreAcudiente: string;
    Vacunas: Vacuna[];

    constructor(){
        this.Vacunas = [];
    }
}

File: Entity/Persona.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;

namespace Entity
{
    public class Persona
    {
        public string TipoDocumento { get; set; }
        [Key]
        public string Cedula { get; set; }
        public string Nombre { get; set; }
        public string Apellido { get; set; }
        public int Edad { get; set; }
        public DateTime FechaNacimiento { get; set; }
        public string InstitucionEducativa { get; set; }
        public string NombreAcudiente { get; set; }
        public List<Vacuna> Vacunas { get; set; }
        
    }
}
